app/src/androidTest/java/com/simplemobiletools/calculator/MoneyActivityTest.java

```diff
@@ -84,6 +85,23 @@ public void taxWithoutGeoLocTest(){
         checkResult("25.99");
     }
 
+    @Test
+    public void testTipDialogDisplayed() {
+        onView(withId(R.id.btn_tip)).perform(click());
+        onView(withText("Calculate Tip")).check(matches(isDisplayed()));
+    }
+
+
+    @Test
+    public void tipOpenApplyTest() {
+        press(R.id.btn_1);
+        press(R.id.btn_0);
+        press(R.id.btn_0);
+        onView(withId(R.id.btn_tip)).perform(click());
```

Comment:
I think a press would work here as well

Reply:
Like I said earlier, that's what I did originally but I couldn't get it to open. idk why. It's only for btn_tip

Reply:
Shouldn't make a difference. you are calling the exact same thing as `press` would

Reply:
Thanks, using 'press' worked. Idk it wasn't working earlier, maybe I made a typo.